### PR TITLE
Bluetooth: VS: First 2 commands

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

File: subsys/bluetooth/controller/hci/hci.c [hunk 1]
```diff
@@ -1490,6 +1494,45 @@ static void vs_read_supported_features(struct net_buf *buf,
...
+		/* Mark IR as invalid */
+		memset(rp->a[0].ir, 0x00, sizeof(rp->a[0].ir));
+
+		rp->status = 0x00;
+
+		return;
+	}
+#endif /* CONFIG_SOC_FAMILY_NRF5 */
+
+	rp->status = BT_HCI_ERR_UNSPECIFIED;
```
Comment (holtmann) [inline subsys/bluetooth/controller/hci/hci.c]: Where is num_addrs handled for the non-nRF5x case. There is really no point in returning an error here. Just return success and an num_addrs = 0. That said, I think the cmd_complete and sizeof(*rp) are not flexible here. You need to able to return num_addrs = 0 and num_addrs = 1.

File: subsys/bluetooth/controller/hci/hci.c [hunk 2]
```diff
@@ -1490,6 +1494,89 @@ static void vs_read_supported_features(struct net_buf *buf,
...
```
Comment (carlescufi) [inline subsys/bluetooth/controller/hci/hci.c]: Not sure if ideal, but couldn't find anything better

File: subsys/bluetooth/controller/hci/hci.c [hunk 3]
```diff
@@ -1490,6 +1494,89 @@ static void vs_read_supported_features(struct net_buf *buf,
...
+	} else {
+		/* Mark ER as invalid */
+		memset(rp->er, 0x00, sizeof(rp->er));
+	}
+
+	rp->status = 0x00;
+	return;
+#endif /* CONFIG_SOC_FAMILY_NRF5 */
+
+	rp->status = BT_HCI_ERR_UNSPECIFIED;
```
Comment (carlescufi) [inline subsys/bluetooth/controller/hci/hci.c]: same as above. alternative is to disable the commands entirely if not on nRF5x

File: subsys/bluetooth/controller/hci/hci.c [hunk 4]
```diff
@@ -1531,6 +1531,50 @@ static void vs_read_static_addrs(struct net_buf *buf, struct net_buf **evt)
...
```
Comment (holtmann) [inline subsys/bluetooth/controller/hci/hci.c]: I don't thing an error appropriate here. Just return IR and ER zeroed out.

**Comment by carlescufi** [comment]
@holtmann comments addressed

**Review by holtmann — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

File: subsys/bluetooth/controller/hci/hci.c [hunk 5]
```diff
@@ -1490,6 +1494,94 @@ static void vs_read_supported_features(struct net_buf *buf,
 	memset(&rp->features[0], 0x00, sizeof(rp->features));
 }
 
+#if defined(CONFIG_BT_CTLR_HCI_VS_EXT)
+static void vs_read_static_addrs(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_rp_vs_read_static_addrs *rp;
+
+#if defined(CONFIG_SOC_FAMILY_NRF5)
```
Comment (cvinayak) [inline subsys/bluetooth/controller/hci/hci.c]: should we have hci_vs_nrfx.c that access h/w registers?

**Pull request merged**